pysb/kappa.py

```diff
@@ -402,26 +410,31 @@ def _parse_kasim_outfile(out_filename):
 
     try:
         out_file = open(out_filename, 'r')
+        preamble_len = 2 # First 2 lines are comments
```

Comment:
I'd suggest looking for the leading `#` on the comments would be more robust than skipping a fixed number of lines, just in case they change the number of headers (I figure the comment character is unlikely to change). E.g.:

```python
for header_count, header_line in enumerate(out_file):
    if header_line[0] != '#':
        break
# No need to call .strip() on header_line before the re.split line
...
arr = np.loadtxt(... skiprows=header_count ...)
```